app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/FoldersGridPresenter.kt

```diff
@@ -34,7 +34,15 @@ class FoldersGridPresenter(
 
     fun init(paths: List<Path>) {
         val folder = findLongestCommonPrefix(paths)
-        frames.addLast(Frame(folder, paths))
+        if (paths.size > 1) {
+            frames.addLast(Frame(folder, paths))
+        } else {
+            val (directories, files) = listChildren(folder)
```

Comment:
So I need to create new branch from main branch. Right ?

Reply:
Yes, that's right. Because commits from each PR might be reworked, so in this case we would need to sync branches, which would unnecessarily spend time.